pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -538,6 +531,52 @@ func createEcDoubleAssignNewXV1Hinter(resolver hintReferenceResolver) (hinter.Hi
 	return newEcDoubleAssignNewXV1Hint(slope, point), nil
 }
 
+// EcDoubleAssignNewYV1 hint computes a new y-coordinate when doubling a point
+// on an elliptic curve
+//
+// `newEcDoubleAssignNewYV1Hint` doesn't take any operander as argument
+// This hint follows the execution of `EcDoubleAssignNewXV1` hint when doubling a point
+// This is why all variables are already accessible in the current scope
+//
+// `newEcDoubleAssignNewYV1Hint` assigns the new y-coordinate as `value` in the current scope
+func newEcDoubleAssignNewYV1Hint() hinter.Hinter {
```

Comment:
This two functions look awfully similar (I only spotted a `x0`, `y0` vs a `x`,`y` in the scope vars). Is there any way to make the two hints the same?

Reply:
No we can't 
This is why I did this refactoring ! To only implement the logic once